<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoreThingMenu</class>
 <widget class="QWidget" name="MoreThingMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>381</width>
       <height>251</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>381</width>
       <height>251</height>
      </size>
     </property>
     <property name="title">
      <string>菜品标签</string>
     </property>
     <widget class="QListWidget" name="lwTags">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>341</width>
        <height>161</height>
       </rect>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>341</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineNewTag"/>
       </item>
       <item>
        <widget class="QPushButton" name="btAddTag">
         <property name="text">
          <string>添加标签</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>381</width>
       <height>461</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>381</width>
       <height>461</height>
      </size>
     </property>
     <property name="title">
      <string>菜品评价</string>
     </property>
     <widget class="QListWidget" name="lwComments">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>341</width>
        <height>241</height>
       </rect>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textComment">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>341</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddComment">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>410</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>添加评价</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="minimumSize">
      <size>
       <width>381</width>
       <height>141</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>381</width>
       <height>141</height>
      </size>
     </property>
     <property name="title">
      <string>评分</string>
     </property>
     <widget class="QPushButton" name="btnCommitStar">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>提交评分</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>34</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="StarButton" name="btnStar1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{background:transparent;border:none;}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Dishes.qrc">
           <normaloff>:/images/star.svg</normaloff>:/images/star.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="StarButton" name="btnStar2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{background:transparent;border:none;}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Dishes.qrc">
           <normaloff>:/images/star.svg</normaloff>:/images/star.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="StarButton" name="btnStar3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{background:transparent;border:none;}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Dishes.qrc">
           <normaloff>:/images/star.svg</normaloff>:/images/star.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="StarButton" name="btnStar4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{background:transparent;border:none;}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Dishes.qrc">
           <normaloff>:/images/star.svg</normaloff>:/images/star.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="StarButton" name="btnStar5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{background:transparent;border:none;}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Dishes.qrc">
           <normaloff>:/images/star.svg</normaloff>:/images/star.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StarButton</class>
   <extends>QPushButton</extends>
   <header location="global">starbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Dishes.qrc"/>
 </resources>
 <connections/>
</ui>
